.github/workflows/interface-unit-tests.yml

```diff
@@ -145,6 +145,8 @@ jobs:
             fromJSON(needs.setup-ci-load.outputs.python-version).torch-tests
             || fromJSON(needs.setup-ci-load.outputs.python-version).default
            }}
+        numpy-2-install: [true]
```

Comment:
Do we want to run `[false, true]` here, or is that too expensive?
Also, shouldn't this be used in every interface test, and not just torch?